test/controllers/qualtrics_controller_test.exs

```diff
@@ -3,9 +3,15 @@ defmodule ResearchResource.QualtricsControllerTest do
 
   setup %{conn: conn} = config do
     if username = config[:login_as] do
-      user = insert_user(email: username)
-      conn = assign(build_conn(), :current_user, user)
-      {:ok, conn: conn, user: user}
+      if username == "survey@completed.com"do
+        user = insert_user(%{email: username, qualtrics_id: "1"})
+        conn = assign(build_conn(), :current_user, user)
+        {:ok, conn: conn, user: user}
+      else
+        user = insert_user(email: username)
+        conn = assign(build_conn(), :current_user, user)
+        {:ok, conn: conn, user: user}
+      end
```

Comment:
Could you take out the common parts of the if/else:

```elixir
if username == "survey@completed.com" do
  user = insert_user(%{email: username, qualtrics_id: "1"})
else
  user = insert_user(%{email: username})
end
conn = assign(build_conn(), :current_user, user)
{:ok, conn: conn, user: user}
```
I find it a little easier to read